src/utils/flagCodes.ts

```diff
@@ -8,7 +8,7 @@ export interface Flag<T extends string> {
 
 // OUR specific flags
 // lowercase hex value to be appended to tx.data
-const SENTINEL = 'dec0de'
+const SENTINEL = CONFIG.transactions.sentinel
```

Comment:
ubernit: Does it make sense to add process.env here so we can change this value for our test servers too for example if we want too?